plugins/chat/assets/javascripts/discourse/components/chat-thread.gjs

```diff
@@ -13,11 +13,14 @@ import { resetIdle } from "discourse/lib/desktop-notifications";
 import { NotificationLevels } from "discourse/lib/notification-levels";
```

Comment:
I would prefer we have all the prompt logic, in a template less component: `<ChatThreadTitlePrompt />`